### ISSUE TITLE
Test classes in default package using ElasticsearchIntegrationTest throws NPE

### ISSUE THREAD

**Issue by stianl** [elastic/elasticsearch#20236]
**Elasticsearch version**: 1.6.2 (same code present on master branch)

**Plugins installed**: []
**JVM version**: NA
**OS version**: NA

**Description of the problem including expected versus actual behavior**:

Test classes in default package that subclasses `ElasticsearchIntegrationTest.java` will throw an NPE in `LoggingListener.java`

**Steps to reproduce**:
 1. Create a test class in the default package that subclasses ElasticSearchIntegrationTest.java
 2. Run test

**Exception**:

```
SEVERE: Panic: RunListener hook shouldn't throw exceptions.
java.lang.NullPointerException
        at org.elasticsearch.test.junit.listeners.LoggingListener.testRunStarted(LoggingListener.java:50)
        at com.carrotsearch.randomizedtesting.RandomizedRunner.runSuite(RandomizedRunner.java:605)
        at com.carrotsearch.randomizedtesting.RandomizedRunner.access$200(RandomizedRunner.java:129)
        at com.carrotsearch.randomizedtesting.RandomizedRunner$1.run(RandomizedRunner.java:559)
```

**LoggingListener.java:50**
```
previousPackageLoggingMap = processTestLogging(description.getTestClass().getPackage().getAnnotation(TestLogging.class));
```

`getPackage()` returns null if the class is not in a package.

**Comment by s1monw** [comment]
can you create a PR for this against master?

**Comment by stianl** [comment]
Yes, I'll do that.

**Comment by jasontedor** [comment]
Closed by #20269
